spec/factories/miq_ae_domain.rb

```diff
@@ -9,6 +10,7 @@
   end
 
   factory :miq_ae_domain, :parent => :miq_ae_namespace do
+    priority 1
```

Comment:
Ok then I think the problem is that the enabled and disabled factories don't inherit from miq_ae_domain properly.  In fact, the name sequencing in those seems wrong...you could create 1 enabled and 1 disabled and they could get the same name.

I think the following is a better layout.  However, at that point I question the need for separate enabled and disabled factories anyway.  For disabled, it's just `FactoryGirl.create(:miq_ae_domain)` and for enabled you just do `FactoryGirl.create(:miq_ae_domain, :enabled => true)`.  Alternatively, if enabled is the bigger use case, then make enabled the default and pass `:enabled => false` where it's needed.

```diff
 FactoryGirl.define do
diff --git a/spec/factories/miq_ae_domain.rb b/spec/factories/miq_ae_domain.rb
index e7a2afe..61a0751 100644
--- a/spec/factories/miq_ae_domain.rb
+++ b/spec/factories/miq_ae_domain.rb
@@ -1,14 +1,15 @@
 FactoryGirl.define do
-  factory :miq_ae_domain_enabled, :parent => :miq_ae_namespace do
-    sequence(:name) { |n| "miq_ae_domain#{seq_padded_for_sorting(n)}" }
+  factory :miq_ae_domain_enabled, :parent => :miq_ae_domain do
     enabled true
   end
 
-  factory :miq_ae_domain_disabled, :parent => :miq_ae_namespace do
-    sequence(:name) { |n| "miq_ae_domain#{seq_padded_for_sorting(n)}" }
+  factory :miq_ae_domain_disabled, :parent => :miq_ae_domain do
   end
 
   factory :miq_ae_domain, :parent => :miq_ae_namespace do
+    sequence(:name) { |n| "miq_ae_domain#{seq_padded_for_sorting(n)}" }
+    priority 1
+
     trait :with_methods do
       transient do
         ae_methods do {'method1' => {:scope => 'instance', :location => 'inline',
```